### PR TITLE
swallow userdel errors

### PR DESCRIPTION
**Pull request by salsal97**

### CODE REVIEW

File: tests/ltp/Makefile [hunk 1]
```diff
@@ -128,12 +128,12 @@ add_host_users_and_groups:
...
 del_host_users_and_groups:
-	sudo userdel user700
-	sudo userdel user702
-	sudo userdel user703
-	sudo groupdel grp701
-	sudo groupdel grp704
-	sudo groupdel grp705
+	sudo userdel user700 || /bin/true
+	sudo userdel user702 || /bin/true
+	sudo userdel user703 || /bin/true
```
Comment (jxyang) [inline tests/ltp/Makefile:133]: Can we just use `sudo userdel user700 || true`?

**Review by jxyang — APPROVED** [review]

File: tests/ltp/Makefile (hunk 1 above)
Comment (salsal97) [reply tests/ltp/Makefile:133]: Hi Xuejun, would it ever act differently?

Comment (jxyang) [reply tests/ltp/Makefile:133]: I think it's the same. We have a few Makefiles use `... || true`

Comment (salsal97) [reply tests/ltp/Makefile:133]: Gotcha, if they act the same I think its okay to keep this here, what do you think?

Comment (jxyang) [reply tests/ltp/Makefile:133]: Yes since this is a blocking issue. Feel free to merge after the pipeline passes.

**Comment by jxyang** [comment]
I have a more complete pipeline fix PR #437

**Comment by salsal97** [comment]
Thanks Xuejun, I'll close this one

**Pull request closed without merging**